SAMv1.tex

```diff
@@ -167,8 +167,7 @@ \subsection{Terminologies and Concepts}
   between the 3rd and the 7th bases inclusive is $[2,7)$. The BAM, BCFv2, BED,
   and PSL formats are using the 0-based coordinate system.
 
-\item[Phred scale] Given a probability $0<p\le 1$, the phred scale of $p$
-  equals $-10\log_{10}p$, rounded to the closest integer.
+\item[Phred scale] Given a probability $0<p\le 1$ of an erroneous call, the phred scale of $p$ equals $-10\log_{10}p$, rounded to the closest integer.
```

Comment:
It seems like the rest of the file is wrapped at 80 characters, do we want to keep that format style with this line?

Reply:
Agreed.  Changing it to match the others.